package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "even-auth",
+    "version": "0.0.1",
+    "description": "Authentication for JS",
```

Comment:
Let's be descriptive like JWT Authentication etc.

Reply:
@hesamd I need to discuss the scope of the project with you before I can write the description.